lib/src/rules/prefer_ints/intl_base_visitor.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:dart_code_metrics/src/utils/object_extensions.dart';
+import 'package:dart_code_metrics/src/utils/iterable_extensions.dart';
+import 'package:meta/meta.dart';
+
+abstract class IntlBaseVisitor extends GeneralizingAstVisitor<void> {
```

Comment:
May be put it in the directory like `intl_base`?

Reply:
moved